services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -53,8 +76,38 @@ class HankeController(
         return hanke
     }
 
-    @GetMapping
-    fun getHankeList(@RequestParam geometry: Boolean = false): List<Hanke> {
+    @GetMapping(produces = [APPLICATION_JSON_VALUE])
+    @Operation(
+        summary = "Get hanke list",
+        description = "Get list of hanke to which the user has permission to."
+    )
+    @ApiResponses(
+        value =
+            [
+                ApiResponse(
+                    description = "Success",
+                    responseCode = "200",
+                    content =
+                        [
+                            Content(
+                                array = ArraySchema(schema = Schema(implementation = Hanke::class))
+                            )
+                        ]
+                )
+            ]
+    )
+    fun getHankeList(
+        @Parameter(
+            description =
+                """Boolean flag indicating whether endpoint should return geometry data for alueet. 
+                    Geometriat fields will be null if false.""",
+            schema = Schema(type = "boolean"),
```

Comment:
Since this parameter is optional, it might be useful to specify what's the default value if the parameter is omitted. Looks like this can be documented with:
```suggestion
            schema = Schema(type = "boolean", defaultValue = "false"),
```

Reply:
Done.